starfish/core/image/Filter/test/test_api_contract.py

```diff
@@ -24,10 +24,26 @@
 
 from starfish import ImageStack
 from starfish.core.image import Filter
-from starfish.core.image._filter.max_proj import MaxProject
-from starfish.core.image._filter.reduce import Reduce
-
-methods: Mapping[str, Type] = Filter._algorithm_to_class_map()
+from starfish.core.image.Filter.max_proj import MaxProject
+from starfish.core.image.Filter.reduce import Reduce
+
+
+methods: Mapping[str, Type] = {
```

Comment:
Can you just do a `dir(Filter)` to get this?

Reply:
ping

Reply:
I tried this and it doesn't quite give me what I need. ```['Bandpass', 'Clip', 'ClipPercentileToZero', 'ClipValueToZero', 'DeconvolvePSF', 'ElementWiseMultiply', 'GaussianHighPass', 'GaussianLowPass', 'Laplace', 'LinearUnmixing', 'MatchHistograms', 'MaxProject', 'MeanHighPass', 'Reduce', 'WhiteTophat', 'ZeroByChannelMagnitude', '__builtins__', '__cached__', '__doc__', '__file__', '__loader__', '__name__', '__package__', '__path__', '__spec__', '_base', 'bandpass', 'clip', 'clip_percentile_to_zero', 'clip_value_to_zero', 'element_wise_mult', 'gaussian_high_pass', 'gaussian_low_pass', 'laplace', 'linear_unmixing', 'match_histograms', 'max_proj', 'mean_high_pass', 'reduce', 'richardson_lucy_deconvolution', 'util', 'white_tophat', 'zero_by_channel_magnitude']``` but I think the refactor to make another clever auto import method will work when I do that!